code/game/objects/structures/false_walls.dm

```diff
@@ -262,22 +262,23 @@
 
 /obj/structure/falsewall/plasma/attackby(obj/item/W, mob/user, params)
 	if(W.is_hot() > 300)
-		var/turf/T = get_turf(src)
-		message_admins("Plasma falsewall ignited by [ADMIN_LOOKUPFLW(user)] in [ADMIN_VERBOSEJMP(T)]")
-		log_game("Plasma falsewall ignited by [key_name(user)] in [AREACOORD(T)]")
-		burnbabyburn()
+		if(plasma_canignite())
+			new /obj/structure/girder/displaced(loc)
+			plasma_ignition(6, user)
```

Comment:
just use the return value of plasma_ignition for if it ignited, and then spawn the girder, rather than having a redundant proc

Reply:
done, feel like an idiot for this one 👍